qa/L0_perf_nomodel/run_test.sh

```diff
@@ -124,6 +139,13 @@ for BACKEND in $BACKENDS; do
         REPO_DIR=./custom_models
     elif [ $BACKEND == "python" ]; then
         REPO_DIR=./python_models
+
+        # FIXME: If PA C API adds SHMEM support, remove this.
+        if [[ "${PERF_CLIENT_PROTOCOL}" == "triton_c_api" ]]; then
```

Comment:
I was wondering why Python backend is excluded here. PA C-API shared memory support is independent from Python backend's internal shared memory.

Reply:
I mis-associated some shm errors I was getting. Thanks for the catch.